test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -728,8 +750,7 @@ BOOST_AUTO_TEST_CASE( eth_sendRawTransaction_errorDuplicateTransaction ) {
     signedTx = fixture.rpcClient->eth_signTransaction( t );
     BOOST_REQUIRE( !signedTx["raw"].empty() );
 
-    BOOST_CHECK_EQUAL( fixture.sendingRawShouldFail( signedTx["raw"].asString() ),
-        "Same transaction already exists in the pending transaction queue." );
+    fixture.sendingRawShouldFail( signedTx["raw"].asString() );

```

Comment:
It's better to check the error message as it was before

Reply:
Fixed